app/src/main/kotlin/com/simplemobiletools/camera/helpers/ImageSaver.kt

```diff
@@ -0,0 +1,304 @@
+package com.simplemobiletools.camera.helpers
+
+import android.annotation.SuppressLint
+import android.content.ContentResolver
+import android.content.ContentValues
+import android.net.Uri
+import android.provider.MediaStore
+import androidx.camera.core.ImageCapture
+import androidx.camera.core.ImageCapture.Metadata
+import androidx.camera.core.ImageCaptureException
+import androidx.camera.core.ImageProxy
+import androidx.camera.core.internal.compat.workaround.ExifRotationAvailability
+import androidx.exifinterface.media.ExifInterface
+import com.simplemobiletools.camera.helpers.ImageUtil.CodecFailedException
+import com.simplemobiletools.camera.helpers.ImageUtil.imageToJpegByteArray
+import com.simplemobiletools.camera.helpers.ImageUtil.jpegImageToJpegByteArray
+import com.simplemobiletools.camera.models.MediaOutput
+import com.simplemobiletools.commons.extensions.copyTo
+import com.simplemobiletools.commons.helpers.ensureBackgroundThread
+import com.simplemobiletools.commons.helpers.isQPlus
+import java.io.*
+import java.util.UUID
+
```

Comment:
avoid such duplicate empty lines please everywhere. Both in code and xml files.

Reply:
Fixed